libfeed/feed-knowledge-app-video-card-store.c

```diff
@@ -0,0 +1,140 @@
+/* Copyright 2018 Endless Mobile, Inc.
+ *
+ * eos-discovery-feed is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * eos-discovery-feed is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with eos-discovery-feed.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+#include "feed-knowledge-app-video-card-store.h"
+#include "feed-sizes.h"
+#include "feed-util.h"
+
+struct _EosDiscoveryFeedKnowledgeAppVideoCardStore
+{
+  GObject parent_instance;
```

Comment:
This isn't right ... the parent instance is `EosDiscoveryFeedKnowledgeAppCardStore` ... I guess this only worked by chance because of the fact that `EosDiscoveryFeedKnowledgeAppCardStore` had `GObject` as a single member too... I'll fix that (same thing applies in a few other places).

Reply:
Hmm, actually, `EosDiscoveryFeedKnowledgeAppCardStore` is an incomplete type unless I define the struct in the header which I'd rather avoid. Since they all only ever contain `GObject` and all the private data is kept in the private pointer section, I suppose this should be fine.